src/transformer/descriptor/tsLibs/typescriptLibsTypeAdapter.ts

```diff
@@ -10,7 +10,9 @@ export function TypescriptLibsTypeAdapter(node: ts.TypeReferenceNode, scope: Sco
   const declaration: ts.Declaration = TypescriptHelper.GetDeclarationFromNode(node.typeName);
   const typeChecker: ts.TypeChecker = TypeChecker();
   const type: ts.Type = typeChecker.getTypeAtLocation(declaration);
-  const typeScriptType: TypescriptLibsTypes = TypescriptLibsTypes[type.symbol.name];
+  const name: string = type.symbol?.name || type.aliasSymbol.name;
```

Comment:
You can use the “safe get aliased symbol” and do .name

It should be somewhere in the helpers

Reply:
thanks, I also forgot to add a test

Reply:
Done, my scenario was a bit different. symbol was undefined.

I've actually found a best way to find the symbol. Have a look.